01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,13 @@
+i = 1
+20.times do
```

Comment:
```suggestion
20.times do |i|
```

timesメソッドは上記のようにブロック引数を追加すると1行目の変数定義と1を足す処理が不要になりますのでそのあたりを調べてみてください～

Reply:
ありがとうございます。
「ブロック引数を追加」バージョンに変更しました。ご確認お願いします。
（前のバージョンは、コメントアウトして残しています。）

Reply:
バージョン管理システムを使っているので基本的にコメントアウトして古いコードを残すことはしないようにするといいかもです。

また例えば

`(1..20).each do |i|`

のようにRangeオブジェクトとeachを使うと`+1`しなくていいのでわかりやすいコードになります。

Reply:
ありがとうございます。
下記２点、変更しました。ご確認お願いします。
１）「`.each`メソッド」バージョンに変更しました。
２）不要な記述を削除しました。
よろしくお願い致します。